From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Thu, 3 Jan 2019 19:37:07 -0500
Subject: universal-access: Remove redundant label-for relations

The appropriate relation is already set with the mnemonic_widget property.

In particular, this also fixes an issue where a screenreader would
emit "Crosshairs" every time a new tab was entered in the Zoom dialog
because of some incorrect labelled-by relations.

(cherry picked from commit 56703258220105101d68a7b2c9f09a907aef59e9)
---
 panels/universal-access/zoom-options.ui | 24 ------------------------
 1 file changed, 24 deletions(-)

diff --git a/panels/universal-access/zoom-options.ui b/panels/universal-access/zoom-options.ui
index 37db882..b8a2ab9 100644
--- a/panels/universal-access/zoom-options.ui
+++ b/panels/universal-access/zoom-options.ui
@@ -131,9 +131,6 @@
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
                 <property name="use_action_appearance">False</property>
-                <accessibility>
-                  <relation type="labelled-by" target="xhairs-section-heading"/>
-                </accessibility>
               </object>
               <packing>
                 <property name="expand">False</property>
@@ -579,9 +576,6 @@
                                     <property name="use_underline">True</property>
                                     <property name="mnemonic_widget">crosshair_length_slider</property>
                                     <property name="width_chars">12</property>
-                                    <accessibility>
-                                      <relation type="label-for" target="crosshair_length_slider"/>
-                                    </accessibility>
                                     <attributes>
                                       <attribute name="scale" value="1.25"/>
                                     </attributes>
@@ -719,9 +713,6 @@
                                 <property name="visible">True</property>
                                 <property name="can_focus">True</property>
                                 <property name="use_action_appearance">False</property>
-                                <accessibility>
-                                  <relation type="labelled-by" target="xhairs-section-heading"/>
-                                </accessibility>
                               </object>
                               <packing>
                                 <property name="expand">False</property>
@@ -832,9 +823,6 @@
                                     <property name="justify">right</property>
                                     <property name="use_underline">True</property>
                                     <property name="mnemonic_widget">inverse_enabled_switch</property>
-                                    <accessibility>
-                                      <relation type="label-for" target="inverse_enabled_switch"/>
-                                    </accessibility>
                                     <attributes>
                                       <attribute name="scale" value="1.25"/>
                                     </attributes>
@@ -856,9 +844,6 @@
                                     <property name="use_underline">True</property>
                                     <property name="mnemonic_widget">brightness_slider</property>
                                     <property name="width_chars">12</property>
-                                    <accessibility>
-                                      <relation type="label-for" target="brightness_slider"/>
-                                    </accessibility>
                                     <attributes>
                                       <attribute name="scale" value="1.25"/>
                                     </attributes>
@@ -879,9 +864,6 @@
                                     <property name="justify">right</property>
                                     <property name="use_underline">True</property>
                                     <property name="mnemonic_widget">contrast_slider</property>
-                                    <accessibility>
-                                      <relation type="label-for" target="contrast_slider"/>
-                                    </accessibility>
                                     <attributes>
                                       <attribute name="scale" value="1.25"/>
                                     </attributes>
@@ -902,9 +884,6 @@
                                     <property name="justify">right</property>
                                     <property name="use_underline">True</property>
                                     <property name="mnemonic_widget">grayscale_slider</property>
-                                    <accessibility>
-                                      <relation type="label-for" target="grayscale_slider"/>
-                                    </accessibility>
                                     <attributes>
                                       <attribute name="scale" value="1.25"/>
                                     </attributes>
@@ -993,9 +972,6 @@
                                         <property name="visible">True</property>
                                         <property name="can_focus">True</property>
                                         <property name="use_action_appearance">False</property>
-                                        <accessibility>
-                                          <relation type="labelled-by" target="xhairs-section-heading"/>
-                                        </accessibility>
                                       </object>
                                       <packing>
                                         <property name="expand">False</property>
